libs/components/forms/src/lib/modules/radio/radio.component.html

```diff
@@ -5,8 +5,8 @@
   <input
     class="sky-radio-input sky-switch-input"
     type="radio"
-    [attr.aria-label]="label"
-    [attr.aria-labelledby]="labelledBy"
+    [attr.aria-label]="labelText || label"
```

Comment:
i believe in this situation if both values are not set ,`aria-label` would still show up in the template. changing this to add ` || null` at the end should fix this